src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -537,10 +537,6 @@ private function findFirstInlinedPrevious(Node $node, callable $filter): ?Node
             return null;
         }
 
-        if ($previousNode === $node) {
-            return null;
-        }
-
```

Comment:
By this PR, the tweak `$previousNode === $node` is no longer needed that previously used to cover infinite loop on find first previous on PR:

- https://github.com/rectorphp/rector-src/pull/3484